bazel/external/requirements_README.md

```diff
@@ -10,7 +10,7 @@ Requirements.txt holds all Python dependencies which are required by Python-base
     
        `cd $MAGMA/bazel/external`
 
-       `pip-compile --generate-hashes --output-file=requirements.txt requirements.in`
+       `pip-compile --upgrade-package --generate-hashes --output-file=requirements.txt requirements.in`
```

Comment:
This is unrelated but I stumbled upon it while working on this PR. The point is that without this parameter, when a `requirements.txt` already exists and the versions satisfy the constraints from `requirements.in`, they are not upgraded. With `--upgrade-package`, versions in an existing requirements.in are upgraded to the highest versions that satisfy the constraints.